src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs

```diff
@@ -224,16 +224,6 @@ private IEnumerable<string> GetSubscriptions()
                 return this.Subscription;
             }
 
-            // Use selected subscription (for example, by command "Select-AzSubscription {subscriptionId}") 
-            if (this.TryGetDefaultContext(out var context))
```

Comment:
@OlhaTkachenko are we able to add a test for this scenario to ensure that we're returning the correct list of subscriptions when the user doesn't provide a value for the `-Subscription` parameter?

Reply:
During test, it is uses subscription from DefaultAccount from SetupAzureEnvironmentFromEnvironmentVariables()